db/const.txt

```diff
@@ -2313,3 +2313,6 @@ ARCWANDCLAN	2
 GOLDENMACECLAN	3
 CROSSBOWCLAN	4
 JUMPINGCLAN	5
+
+IG_PrizeOfHero	453
+IG_Squad_Prize	456
```

Comment:
Please move those over to https://github.com/rathena/rathena/blob/master/src/map/itemdb.h#L728 and export them via  https://github.com/rathena/rathena/blob/master/src/map/script_constants.h#L3814

Thank you!

Reply:
Okay, I didn't update document for a long time 😁